docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -222,12 +593,60 @@ The following table lists all the server state attributes the client can receive
 | `MC_PLAYBACK_STATE_PREV_FILE`    | Playback state of previous file          |
 | `MC_PLAYBACK_STATE_FAST_FORWARD` | Playback state of fast forward           |
 | `MC_PLAYBACK_STATE_REWIND`       | Playback state of rewind                 |
-| **Shuffle mode states**          |                                          |
+| `MC_PLAYBACK_STATE_MOVING_TO_NEXT` | Playback state of moving to next media (Tizen 4.0 and higher for Mobile and Tizen 5.0 and Higher for Wearable) |
+| `MC_PLAYBACK_STATE_MOVING_TO_PREVIOUS` | Playback state of moving to previous media (Tizen 4.0 and higher for Mobile and Tizen 5.0 and Higher for Wearable) |
+| `MC_PLAYBACK_STATE_FAST_FORWARDING` | Playback state of fast forwarding (Tizen 4.0 and higher for Mobile and Tizen 5.0 and Higher for Wearable) |
+| `MC_PLAYBACK_STATE_REWIND`       | Playback state of rewinding (Tizen 4.0 and higher for Mobile and Tizen 5.0 and Higher for Wearable) |
+
+<a name="playbackaction"></a>
```

Comment:
As mentioned earlier, please avoid using HTML code. Use only MD specific code.

Reply:
The 'playbackaction' tag is not used. So I removed it.